lib/qr_code_scanner/lib/src/overlay/scan_area_indicator_overlay.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+/// A widget that indicates a scan area in the middle of the screen.
+///
+/// Technically, everything that is covered by the camera could be scanned. The
+/// [ScanAreaIndicatorOverlay] is just a visual indicator for the user.
+class ScanAreaIndicatorOverlay extends StatelessWidget {
+  const ScanAreaIndicatorOverlay({
+    Key? key,
+    this.color = Colors.white,
+    this.width = 250,
+  }) : super(key: key);
+
+  /// The color of the scan area corners.
+  ///
+  /// Defaults to [Colors.white].
+  final Color color;
+
+  /// The width of the visual scan area.
+  ///
+  /// Defaults to 250.
+  final double width;
```

Comment:
Is `width` the right word? Isn't this more like "size" but specified by giving the length of the square?    
I'm just saying that one might think that if you increase width then only the width but not the height should be increasing.